framework/composer.json

```diff
@@ -54,7 +54,6 @@
     "require": {
         "php": ">=5.4.0",
         "ext-mbstring": "*",
-        "ext-mcrypt": "*",
```

Comment:
what do you mean with that? it is a php extension the same way as mcrypt is one... http://php.net/manual/en/mcrypt.installation.php